UIKitVK/UIKitVK/Model/Group/Group.swift

```diff
@@ -6,13 +6,17 @@ import RealmSwift
 
 // Группы
 final class Group: Object, Codable {
-    @objc dynamic var id: Int
-    @objc dynamic var name: String
-    @objc dynamic var groupAvatar: String?
+    @Persisted var id: Int
+    @Persisted var name: String
+    @Persisted var groupAvatar: String?
 
     private enum CodingKeys: String, CodingKey {
         case id
         case name
         case groupAvatar = "photo_50"
     }
+
+    override static func primaryKey() -> String? {
+        "id"
+    }
 }
```

Comment:
```suggestion
    @Persisted(primaryKey: true) var id: Int
    @Persisted var name: String
    @Persisted var groupAvatar: String?

    private enum CodingKeys: String, CodingKey {
        case id
        case name
        case groupAvatar = "photo_50"
    }
}
```

если делать через персайтед то можно назначить главный ключ, в другом месте ты так и делае